scripts/functions/transform.R

```diff
@@ -205,20 +204,32 @@ transform <- function(full_load = TRUE){
   # 4. Speichere Daten als CSV und Zip
   ##---------------------------------------------------------------------------
 
-  # Speichere aufbereites messwerte File als CSV
-  readr::write_csv(df_final, file = paste0(path_to_load_folder, aufbereite_messwerte_ogd_filename, ".csv"))
+  cli::cli_alert_info("Speichere Rohdaten lokal:")
+  #Erstelle temporäres Verzeichnis und speichere CSV in Verzeichnis. Dieses Verzeichnis wird danach gezippt
+  dir.create(paste0(path_to_load_folder, rohdaten_messwerte_ogd_filename))
```

Comment:
```suggestion
  dir.create(paste0(path_to_load_folder, rohdaten_messwerte_ogd_filename), recursive = TRUE, showWarnings = FALSE)
```
Könnte man einbauen damit das Directory immer gebaut wird, auch wenn Top-Folder fehlt. Fehlermeldung/Abbruch ohne die Argumente sollte aber ohnehin ausgeschlossen sein, da die Pfade gesetzt und benötigt werden. Überlasse es dir :)